documentation/docs/components/Cognite3DViewerDemo.tsx

```diff
@@ -35,17 +35,15 @@ export default function Cognite3DViewerDemo({
       antiAliasingHint: 'msaa4+fxaa'
     });
 
-    async function addModel(options: AddModelOptions) {
-      const model = await viewer.addModel(options);
+    viewer.addModel({ modelId, revisionId }).then((model) => {
       viewer.loadCameraFromModel(model);
-      window.model = model;
```

Comment:
Don't we still need to add the model to the `window`?

Reply:
I think it was only for debug purposes, if it is needed, can be left there.

Reply:
I'd prefer to remove it

Reply:
Also, you can access it through `viewer.models` easily.

Reply:
We use `window.model` a lot in the examples. That's how we inject the model into the example snippets: https://github.com/cognitedata/reveal/blob/10e7062654ba4260848ccd022a67c7d5f90758d6/documentation/docs/components/LiveCodeSnippet.tsx#L59 E.g. in the `Loading CAD models` page, the first example tries to hide the model, but the model stays visible.